src/main/resources/data/celtic_rituals/recipes/healing_ritual.json

```diff
@@ -0,0 +1,14 @@
+{
+  "type": "celtic_rituals:bowl_ritual",
+  "ingredients": [
+    {
+      "item": "minecraft:redstone"
+    },
+    {
+      "item": "minecraft:gold_nugget"
+    }
+  ],
+  "result": {
+    "item": "minecraft:glowstone_dust"
+  }
+}
```

Comment:
Please an empty line at the end

Reply:
This should also be done by a linter. See #21